frontend/src/toolbar/stats/HeatmapStat.scss

```diff
@@ -0,0 +1,3 @@
+.HeatmapStats__elements-list:hover {
+    background-color: var(--primary-highlight);
+}
```

Comment:
This is totally fine, but if I were to create more work, I'd suggest adding some SCSS magic to autogenerate a bunch of `hover:bg-primary` [tailwindesque](https://tailwindcss.com/docs/background-color#hover-focus-and-other-states) color classes in utilities.scss 🙃

Reply:
OMG I opened the file, pressed enter and github co-pilot suggested

```
@each $name, $hex in $colors {
    .hover\\:text-#{$name}:hover {
        color: $hex;
    }
    .hover\\:bg-#{$name}:hover {
        background-color: $hex;
    }
    .hover\\:border-#{$name}:hover {
        border-color: $hex;
    }
}
```

How does it know?!

https://github.com/PostHog/posthog/pull/13870

Reply:
The time AI saves is offset by the time developers are amazed at how much time AI saves

Reply:
me: *saves 4 seconds using co-pilot
also me: shares screenshot in slack, mastodon, twitter, and github UI taking 7 minutes